docs/custombuilders.md

```diff
@@ -0,0 +1,113 @@
+# Experimental Custom Builders
+
+kpack provides the experimental CustomBuilder and CustomClusterBuilder resources to define and create [Cloud Native Buildpacks builders](https://buildpacks.io/docs/using-pack/working-with-builders/) all within the kpack api. 
+This makes it easy to quickly configure and create builders and use them with Image Resources. Before creating CustomBuilders you will need to create the Stack and Store resources described below. 
+
+### <a id='store'></a>Store
+
+The store is a cluster scoped resource that is configured with a list of images that contain buildpacks that are are available to be used in a CustomBuilder.
+
+```yaml
+apiVersion: experimental.kpack.pivotal.io/v1alpha1
+kind: Store
+metadata:
+  name: sample-store
+spec:
+  sources:
+  - image: gcr.io/cf-build-service-public/node-engine-buildpackage@sha256:95ff756f0ef0e026440a8523f4bab02fd8b45dc1a8a3a7ba063cefdba5cb9493
+  - image: gcr.io/cf-build-service-public/npm-buildpackage@sha256:5058ceb9a562ec647ea5a41008b0d11e32a56e13e8c9ec20c4db63d220373e33
+  - image: cloudfoundry/cnb:bionic
+```
+
+For kpack to use an image in the store it must contain buildpacks and buildpack metadata in `io.buildpacks.buildpack.layers` label. Buildpackages and existing builders are commonly used.     
+ 
+### <a id='store'></a>Stack
+
+The Stack is a cluster scoped resource that provides the configuration for a [Cloud Native Buildpack stack](https://buildpacks.io/docs/concepts/components/stack/) that is available to be used in a Custom Builder.    
+
+```yaml
+apiVersion: experimental.kpack.pivotal.io/v1alpha1
+kind: Stack
+metadata:
+  name: bionic-stack
+spec:
+  id: "io.buildpacks.stacks.bionic"
+  buildImage:
+    image: "cloudfoundry/build@sha256:b30b850f5b4d2e11313f0ec152952eace285ce7a3bc203ca5cdcfa8e5bb232a6"
+  runImage:
+    image: "cloudfoundry/run@sha256:ba9998ae4bb32ab43a7966c537aa1be153092ab0c7536eeef63bcd6336cbd0db"
+```
+
+### <a id='store'></a>Custom Builders
+
+The CustomBuilder utilize a [Store](#store) and a [Stack](#stack) and a order definition to construct a builder image. 
```

Comment:
Nitpick:
`The CustomBuilder uses a [Store](#store), a [Stack](#stack), and an order definition to construct a builder image.`

Reply:
Love it.